kolibri/plugins/facility/assets/src/views/users/UsersRootPage/index.vue

```diff
@@ -279,10 +278,6 @@
         fetchClasses();
       });
 
-      function clearSelectedUsers() {
-        selectedUsers.value = new Set();
-      }
-
```

Comment:
This is also present in the new user and trash pages and could likewise be removed in favor of your new `clearSelectedUsers` method returned from `useUserManagement`

Reply:
Got it!